starter-code/app.js

```diff
@@ -9,6 +11,23 @@ const mongoose     = require('mongoose');
 const logger       = require('morgan');
 const path         = require('path');
 
+// TEST 
+// var Recaptcha = require('express-recaptcha').RecaptchaV3;
+// var recaptcha = new Recaptcha('SITE_KEY', 'SECRET_KEY',{callback:'cb'});
+// END TEST
+
+const bcrypt     = require("bcrypt");
+const saltRounds = 10;
+
+const plainPassword1 = "HelloWorld";
+const plainPassword2 = "helloworld";
+
+const salt  = bcrypt.genSaltSync(saltRounds);
+const hash1 = bcrypt.hashSync(plainPassword1, salt);
+const hash2 = bcrypt.hashSync(plainPassword2, salt);
+
+console.log("Hash 1 -", hash1);
+console.log("Hash 2 -", hash2);
 
```

Comment:
I recommend removing your tests from your code after you're finished, it's better not to have unnecessary code laying around your project (in this case line 14-30 aren't needed)